app/Svg/Generator.hs

```diff
@@ -263,6 +266,7 @@ edgeToSVG styled path =
     S.path ! A.id_ (textValue . T.append "path" . pathId_ $ path)
            ! A.class_ "path"
            ! A.d (textValue . T.cons 'M' . buildPathString . pathPoints $ path)
+           ! A.transform (stringValue . show . formatTransform $ pathTransform path)
```

Comment:
I am unsure this is the right place to add this line since it is not included in a <g> tag.